modules/cli-developer-troubleshooting.adoc

```diff
@@ -35,6 +35,21 @@ Launch a command shell to debug a running application.
 $ oc debug deploymentconfig/python
 ----
 
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+
+.Example: Debug the `node1` node
+[source,terminal]
+----
+$ oc debug node/node1
+----
+
+[NOTE]
+====
+If the scheduler has a default node selector set, the `oc debug node/<node>` command might not succeed. To work around this, first create a temporary project using `oc adm new-project` that overwrites the default node selector. Then, run the `oc debug node/<node>` command in that temporary project.
```

Comment:
Put the entire command here `oc adm new-project <temporary_project> --node-selector=""` and link to https://docs.openshift.com/container-platform/4.5/nodes/scheduling/nodes-scheduler-default.html as well